### PR TITLE
fix: hotfix for catering a specific org on cybermonday

### PR DESCRIPTION
**Pull request by uzairr**
ES is behaving weird in case of filtering the courses against an org.This PR introduces a hotfix which will mask a particular org to not been checked against ES.

### CODE REVIEW

File: course_discovery/apps/api/v1/views/catalog_queries.py [hunk 1]
```diff
@@ -46,6 +46,14 @@ def get(self, request):
...
                 course_uuids = [UUID(course_uuid) for course_uuid in course_uuids.split(',')]
                 specified_course_ids += course_uuids
+                # hotfix for Cyber Monday starting (28th Nov 2022)
+                courses_orgs = [
+                    Course.objects.get(uuid=course_uuid).key.split('+')[0] for course_uuid in specified_course_ids
+                ]
+                if 'GTx' not in courses_orgs:
+                    contains = {str(course_uuid): bool(course_uuid) for course_uuid in specified_course_ids}
+                    return Response(contains)
+
```
Comment (colinbrash) [inline course_discovery/apps/api/v1/views/catalog_queries.py:58]: This endpoint is used for a lot more than just for this coupon.  So we need to make sure this logic doesn't get applied other API queries.  (If it did, **every single** API query, including `org:(GTx)` would return `true` if the courses were not GTx!

Comment (colinbrash) [reply course_discovery/apps/api/v1/views/catalog_queries.py:58]: So -- you need to check the `query`.  Only apply this logic if it is equal to `number:(-CSE6040x AND -ISYE6501x AND -MGT6203x) AND org:(-GTx)`

File: course_discovery/apps/api/v1/views/catalog_queries.py [hunk 2]
```diff
@@ -46,6 +46,14 @@ def get(self, request):
             if course_uuids:
                 course_uuids = [UUID(course_uuid) for course_uuid in course_uuids.split(',')]
                 specified_course_ids += course_uuids
+                # hotfix for Cyber Monday starting (28th Nov 2022)
+                courses_orgs = [
+                    Course.objects.get(uuid=course_uuid).key.split('+')[0] for course_uuid in specified_course_ids
+                ]
+                if 'GTx' not in courses_orgs:
+                    contains = {str(course_uuid): bool(course_uuid) for course_uuid in specified_course_ids}
```
Comment (colinbrash) [inline course_discovery/apps/api/v1/views/catalog_queries.py]: Instead of `bool(course_uuid)` you just want `true`.

**Review by colinbrash — CHANGES_REQUESTED** [review]

File: course_discovery/apps/api/v1/views/catalog_queries.py (hunk 2 above)
Comment (colinbrash) [reply course_discovery/apps/api/v1/views/catalog_queries.py]: Also would be a good idea to add logs, so we can be more confident it is working as intended!

Comment (uzairr) [reply course_discovery/apps/api/v1/views/catalog_queries.py]: it will evaluate to `true`, putting it directly is raising error.

**Review by colinbrash — APPROVED** [review]
Looks good.  Will Slack you about testing!

**Review by ansabgillani — APPROVED** [review]

**Pull request merged**